src/components/BookForm.js

```diff
@@ -1,16 +1,31 @@
-const BookForm = () => (
-  <div className="form__container">
-    <form>
-      <h2>ADD NEW BOOK</h2>
-      <div className="form__group">
-        <input type="text" name="title" placeholder="Book title" />
-      </div>
-      <div className="form__group">
-        <input type="text" name="author" placeholder="Book author" />
-      </div>
-      <button type="submit">ADD BOOK</button>
-    </form>
-  </div>
-);
+import { useDispatch } from 'react-redux';
+import { addBooks } from '../redux/books/books';
 
+const BookForm = () => {
+  // const inputValue = useSelector((state) => (state.books));
+  const dispatch = useDispatch();
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    dispatch(addBooks(e.target.title.value, e.target.author.value));
+  };
+
+  return (
+    <div className="form__container">
+      <form onSubmit={handleSubmit}>
+        <h2>ADD NEW BOOK</h2>
+        <div className="form__group">
+          <input type="text" name="title" placeholder="Book title" />
+        </div>
+        <div className="form__group">
+          <input type="text" name="author" placeholder="Book author" />
+        </div>
+        <button type="submit">
+          ADD BOOK
+        </button>
+      </form>
+    </div>
+  );
+};
```

Comment:
- [ ] Kindly use the local state for your `title` and `author` inputs as user inputs should be controlled. The local state should be updated using the `onChange` listener. You can visit [this article](https://dmitripavlutin.com/controlled-inputs-using-react-hooks/) on how to do that.